UnityProject/Assets/Scripts/Systems/Inventory/Inventory.cs

```diff
@@ -130,12 +131,12 @@ public static bool ServerDrop(ItemSlot fromSlot, Vector2? worldTargetVector = nu
 	/// <returns>true if successful</returns>
 	public static void ServerDropAll(DynamicItemStorage playerStorage, Vector2? worldTargetVector = null)
 	{
-		var playerItems = playerStorage.GetItemSlots();
+		var playerItems = playerStorage.GetItemSlots().ToList();
```

Comment:
what type does ``GetItemSlots`` return that you need to cast to list?

Reply:
IEnumerable

Reply:
by recent discussion on github, I think ``Count`` works on that

Reply:
wdym?

Reply:
meaning you might not need to cast to list

Reply:
you can't `EnumerableList[index]`